sdk/testing/src/main/java/io/opentelemetry/sdk/testing/assertj/OpenTelemetryAssertions.java

```diff
@@ -26,5 +26,5 @@ public static SpanDataAssert assertThat(SpanData spanData) {
     return new SpanDataAssert(spanData);
   }
 
-  protected OpenTelemetryAssertions() {}
+  private OpenTelemetryAssertions() {}
```

Comment:
@bogdandrutu @jkwatson Yeah it's normal practice for the entrypoint into assertions to be extendable - it's how a user would be able to add their own custom assertions while using these too.

Reply:
I'm totally fine opening these back up. It's testing code, not production code, and I think we should make it as easy as possible for our users to write tests for otel and extend as needed.